.github/workflows/sam-build-tag-publish-and-run-tests.yaml

```diff
@@ -125,7 +125,7 @@ jobs:
       - name: dispatch to terra-github-workflows
         uses: aurelien-baudet/workflow-dispatch@93e95b157d791ae7f42aef8f8a0d3d723eba1c31 #commit sha for v2.1.1
```

Comment:
Replace `aurelien-baudet/workflow-dispatch@93e95b157d791ae7f42aef8f8a0d3d723eba1c31 #commit sha for v2.1.1` with the newer `broadinstitute/workflow-dispatch@v3`. The newer version is less buggy and avoid using deprecated GitHub library functions.